HACKING.rst.txt

```diff
@@ -286,7 +286,7 @@ improvements, and submit your first PR!
 
 .. _Pytest: https://pytest.org/
 
-.. _mailing lists: https://www.scipy.org/mailing-lists/
+.. _mailing lists: https://projects.scipy.org/scipylib/mailing-lists.html
```

Comment:
This is an old website we don't want to refer people to. Only scipy.org should be used moving forward.
```suggestion
.. _mailing lists: https://scipy.org/community/#scipy-mailing-list
```